examples/rbd/pvc-block-clone.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: block-pvc-clone
+spec:
+  storageClassName: csi-rbd-sc
+  volumeMode: Block
+  dataSource:
+    name: raw-block-pvc
```

Comment:
please just use `block-pvc` like `volumeMode: Block` (it does not have `raw` in the value)

Reply:
yeah.. I will remove the  entire `raw` strings in next PR.